tools/UpdatePaths/clothingunderrepath.txt

```diff
@@ -183,8 +183,8 @@
 /obj/item/clothing/under/rank/warden/navyblue : /obj/item/clothing/under/rank/security/warden/formal
 /obj/item/clothing/under/rank/det : /obj/item/clothing/under/rank/security/detective
 /obj/item/clothing/under/rank/det/skirt : /obj/item/clothing/under/rank/security/detective/skirt
-/obj/item/clothing/under/rank/det/grey : /obj/item/clothing/under/rank/security/detective/grey
-/obj/item/clothing/under/rank/det/grey/skirt : /obj/item/clothing/under/rank/security/detective/grey/skirt
+/obj/item/clothing/under/rank/det/grey : /obj/item/clothing/under/rank/security/detective/noir
+/obj/item/clothing/under/rank/det/grey/skirt : /obj/item/clothing/under/rank/security/detective/noir/skirt
```

Comment:
Include update paths for `/obj/item/clothing/under/rank/security/detective/grey` and `/obj/item/clothing/under/rank/security/detective/grey/skirt` too.

Reply:
Why, if you don't mind me asking? Not much of a coder, I was just changing paths.